crates/sui-indexer-alt/src/handlers/kv_objects.rs

```diff
@@ -20,34 +21,30 @@ impl Processor for KvObjects {
     const NAME: &'static str = "kv_objects";
     type Value = StoredObject;
 
-    async fn process(&self, checkpoint: &Arc<CheckpointData>) -> Result<Vec<Self::Value>> {
-        let deleted_objects = checkpoint
-            .eventually_removed_object_refs_post_version()
-            .into_iter()
-            .map(|(id, version, _)| {
+    async fn process(&self, checkpoint: &Arc<Checkpoint>) -> Result<Vec<Self::Value>> {
+        let deleted_objects = checkpoint.transactions.iter().flat_map(|txn| {
```

Comment:
This seems to change the semantics quite a bit. On the left, `eventually_removed_object_refs_post_version` tracks objects that are deleted/ wrapped/unwrapped_then_deleted in the checkpoint but not recreated (unwrapped) later in the same checkpoint. But on the right we gather all the  deleted objects in all transactions and miss the wrap/unwrap scenarios. Maybe it's best to keep the semantics the same for the scope of this PR?